server/routes/api/messages.js

```diff
@@ -43,4 +51,88 @@ router.post("/", async (req, res, next) => {
   }
 });
 
+// Update mssage read status in conversation
+router.patch("/", async (req, res, next) => {
+  try {
+    if (!req.user) {
+      return res.sendStatus(401);
+    }
+
+    const { senderId, activeConversationId } = req.body;
+
+    // check if activeConversationId is exist
+    if (activeConversationId) {
+      let conversation = await Conversation.findConversationById(
+        activeConversationId,
+        senderId,
+        req.user.id
+      );
+
+      // if not exist return error
+      if (!conversation) return res.json(conversation);
```

Comment:
Here you should be returning an explicit http status code (and optional error message) - in this case a 400 Bad Request would fit since they sent a bad conversationId